src/CommunityToolkit.Maui.MediaElement/MediaElement.shared.cs

```diff
@@ -372,14 +371,12 @@ public void SeekTo(TimeSpan position)
 	/// <inheritdoc cref="IMediaElement.Stop"/>
 	public void Stop()
 	{
-		ClearTimer();
 		OnStopRequested();
 		Handler?.Invoke(nameof(StopRequested));
 	}
 
 	internal void OnMediaEnded()
 	{
-		ClearTimer();
```

Comment:
Why not clear the time when the media ended?

Reply:
`OnMediaEnded` will occur when you drag the progress-bar to the end (or play to the end)... very annoying that you need to reload the media (it's actually still loaded) after hitting the progress-bar end, in order to get progress-bar working again. 
Neither libmpv or libvlc works this way.

Starting the timer when loading the media, and stopping it when opening a new media (or setting it to null) makes sense to me.

Developers can use `OnMediaEnded` to unload/null the media file/stream if they really want the media closed and the progress-bar unresponsive.

Reply:
@pictos: Pedro, could you give this a look - it would really help, so I will not need to reload the video after moving the progress bar to the end or playing the last part of the video. It's also how the video players I normally use works (libmpv, libvlc, directshow, mpc). I could really need in order to port this Winforms project to Mai - https://github.com/SubtitleEdit/subtitleedit
TIA :)